build.sbt

```diff
@@ -146,9 +146,13 @@ lazy val site = project
   .settings(
     name := "sbt-typelevel-site"
   )
-  .dependsOn(kernel, githubActions, noPublish)
+  .dependsOn(kernel, github, githubActions, noPublish)
 
 lazy val docs = project
   .in(file("mdocs"))
   .enablePlugins(TypelevelSitePlugin)
-  .settings(laikaConfig ~= { _.withRawContent })
+  .enablePlugins(ScalaUnidocPlugin)
+  .settings(
+    tlSiteApiUri := Some(uri("api/org/typelevel/sbt/")),
```

Comment:
Overriding this so we are teleported to the right package in the docs. Seems hard to configure automatically, but it's only a nice-to-have, not critical.